scvi/autotune/_manager.py

```diff
@@ -86,31 +88,58 @@ def _cls_to_tunable_type(cls: Any) -> str:
                     return tunable_type
             return "unknown"
 
+        def _parse_func_params(func: Callable, parent: Any, tunable_type: str) -> dict:
+            # parse function docstring for parameter descriptions
+            docstring = None
+            if func.__name__ == "__init__":
+                # __init__ func docstrings are the class's docstring
+                docstring = parse(parent.__doc__)
```

Comment:
I think we might be able to use docrep for this instead of introducing another dependency?

Reply:
Might just remove the functionality to print out keyword argument descriptions, I think it's too overloaded with it.